packages/@react-aria/overlays/test/useOverlay.test.js

```diff
@@ -127,10 +147,90 @@ describe('useOverlay', function () {
     installPointerEvent();
     it('should prevent default on pointer down on the underlay', function () {
       let underlayRef = React.createRef();
-      render(<Example isOpen isDismissable underlayProps={{ref: underlayRef}} />);
-      let isPrevented = fireEvent.pointerDown(underlayRef.current, {button: 0, pointerId: 1});
+      render(
+        <Example isOpen isDismissable underlayProps={{ref: underlayRef}} />
+      );
+      let isPrevented = fireEvent.pointerDown(underlayRef.current, {
+        button: 0,
+        pointerId: 1
+      });
       fireEvent.pointerUp(document.body);
       expect(isPrevented).toBeFalsy(); // meaning the event had preventDefault called
     });
   });
 });
+
+describe('useOverlay with shadow dom', () => {
```

Comment:
I don't think these tests are going to be enough, usually overlays portal out to be a direct child of the `body` element, which would actually take them out of a shadowroot potentially

This can also be changed with UNSTABLE_PortalProvider

We'll need tests using the RAC Popover at a minimum